### DISCUSSION TITLE
False positive on manga

### DISCUSSION THREAD

**Discussion by Namchee** [infinitered/nsfwjs#706] [Model Training]
## Overview

Model used: Inception v3. MobileNet models does not experience this issue.

![ar0ltbti00h71](https://user-images.githubusercontent.com/32661241/145770552-f7b8c11d-7d73-43d7-a04f-97b3315c6eee.jpg)

![proof](https://user-images.githubusercontent.com/32661241/145770838-b8b0fada-d9d0-437a-bbb9-88a021906cd1.png)

**Comment by GantMan** [comment]
I'm thinking about making a voting layer that allows you to combine multiple models and get higher accuracy.
